style.css

```diff
@@ -24,10 +33,25 @@ section {
   border-radius: 10%;
   margin-top: 100px;
   margin-bottom: 100px;
+  margin-right: 100px;
+  margin-left: 100px;
```

Comment:
All four margins have the same value, use margin: 100px; as a single line instead of four separate lines.